.gitignore

```diff
@@ -1,4 +1,5 @@
 .idea
 *.iml
 /target
-deploy.sh
\ No newline at end of file
+deploy.sh
+work/*
```

Comment:
??

Reply:
when you start with mvn hpi:run, it creates this work directory. I can't think of a situation where you would want to not ignore this...

Reply:
Ok sounds, great. Did not do this before.